cms/djangoapps/contentstore/views/component.py

```diff
@@ -40,7 +40,7 @@
 log = logging.getLogger(__name__)
 
 # NOTE: This list is disjoint from ADVANCED_COMPONENT_TYPES
-COMPONENT_TYPES = ['discussion', 'html', 'problem', 'video']
+COMPONENT_TYPES = ['discussion', 'html', 'problem', 'openassessment', 'video']
```

Comment:
In the requirement's mockups, this appears before the `problem` button, can you change the order here?
![ORA-template](https://user-images.githubusercontent.com/27893385/93931560-574be480-fcf5-11ea-80ff-10ad7d128ff6.png)

Reply:
Makes sense will do it :)